index.html

```diff
@@ -21,6 +21,17 @@ <h1>RustFest {{site.location.city}} {{site.dates.year}}</h1>
           {{site.dates.workshops}}<br />
         </p>
       </div>
+
+      <div>
+        <p class="desc">Impl Days</p>
+        <p>
+          {{site.dates.impl_days}}, {{site.dates.year}}
+          <p class="note">
+            <a href="/about_impl_days/">What are the <code class="highlighter-rouge">impl Days</code>?</a>
```

Comment:
The Wording is off here, either "impl days" or "Impl Days" but stick with one.

Reply:
`impl Days` is the correct spelling but I was capitalizing since it is a "title" (sorry about that, I'm not a copy and it shows!) so I'll change it.

Which - by the way - won't make any difference since this text is fully CAPITALIZED via css :-)